packages/@lwc/babel-plugin-component/src/decorators/index.ts

```diff
@@ -143,7 +141,7 @@ function isImportedFromLwcSource(decoratorBinding: Binding) {
 function validate(decorators: DecoratorMeta[], state: LwcBabelPluginPass) {
     for (const { name, path } of decorators) {
         const binding = path.scope.getBinding(name);
-        if (binding === undefined || !isImportedFromLwcSource(binding)) {
+        if (binding === undefined || !isImportedFromLwcSource(binding.path)) {
             throw generateInvalidDecoratorError(path, state);
         }
     }
```

Comment:
This is the only non-type change I kept. Referencing the `Binding` type would be too awkward via `NodePath['scope'][string]` and the function only uses its path property which is already imported.